news/10410.feature.rst

```diff
@@ -0,0 +1,2 @@
+Allways fallback to reporting the editable project location when pip freeze
```

Comment:
typo:

```suggestion
Always fallback to reporting the editable project location when pip freeze
```

Reply:
nit-ish: I would rephrase this to something like:

> `pip freeze` will now ...

I find it easier to skim, when we mention the "thing-that-we-affected" at the start of the message.